src/woo_publications/api/models/category.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ordered_model.models import OrderedModel
+
+from ..constants import InformatieCategorieChoices
+
+
+class InformatieCategorie(OrderedModel):
+    identifier = models.URLField(_("Identifier"), max_length=255, unique=True)
+    name = models.CharField(_("Name"), max_length=80)
+    name_plural = models.CharField(_("Name plural"), max_length=80)
+    definition = models.TextField(_("Definition"), blank=True)
+    origin = models.CharField(
+        _("Origin"),
+        choices=InformatieCategorieChoices.choices,
+        blank=False,
+        max_length=15,
+    )
+
+    class Meta:
+        verbose_name = _("informatie categorie")
+        verbose_name_plural = _("informatie categorieën")
+        ordering = ("order",)
```

Comment:
```suggestion
    class Meta(OrderedModel.Meta):
        verbose_name = _("informatie categorie")
        verbose_name_plural = _("information categories")
```
* the `Meta` inheritance takes care of setting up the ordering properly, it's a shame they don't document this properly.
* Code base must be English - the only exception is stuff that directly ends up in the API spec since we want that interface to be in Dutch to avoid people bikeshedding about that. Now I'm not sure if we should define the model fields in English or not - I *think* at the API level you can use the serializer `source` option via `Serializer.Meta.extra_kwargs` but if that is not an obvious solution, let's just use Dutch field names